.eslintrc.js

```diff
@@ -1,185 +1,18 @@
-                    },
-                    {
-                        pattern: '@/**',
-                        group: 'internal',
-                    },
-                ],
-            },
-        ],
-        'import/prefer-default-export': 'off',
-        'import/no-extraneous-dependencies': 'error',
-        'sort-imports': [
-            'error',
-            {
-                ignoreCase: true,
-                ignoreDeclarationSort: true,
-                ignoreMemberSort: false,
-                memberSyntaxSortOrder: ['none', 'all', 'multiple', 'single'],
-            },
-        ],
-
-        // General code-style rules
-        '@typescript-eslint/naming-convention': [
-            'warn',
-            {
-                selector: 'default',
-                format: ['snake_case', 'strictCamelCase', 'StrictPascalCase', 'UPPER_CASE'], // snake_case enabled for Slate types
-                // leadingUnderscore is allowed to allow having unused leading function arguments, e.g.
-                // const ownPropSelector = (_state, ownProps) => ownProps.id;
-                leadingUnderscore: 'allow',
-                trailingUnderscore: 'forbid',
-            },
-            {
-                // This is due to backend sending us snake_case properties
-                selector: 'property',
-                format: ['snake_case', 'strictCamelCase', 'StrictPascalCase', 'UPPER_CASE'],
-                leadingUnderscore: 'forbid',
-                trailingUnderscore: 'forbid',
-            },
-            {
-                // This is due to Algolia "_tags", "_highlightResult", "_snippetResult" & "objectID" properties
-                // Also for Airtable's "baseId"
-                selector: 'property',
-                format: ['camelCase'],
-                filter: {
-                    regex: '^_?(?:tags|highlightResult|snippetResult|objectID|baseID)$',
-                    match: true,
-                },
-                leadingUnderscore: 'allow',
-                trailingUnderscore: 'forbid',
-            },
-            {
-                // This is for `dangerouslySetInnerHTML`
-                selector: 'property',
-                format: ['camelCase'],
-                filter: {
-                    regex: '^__html$',
-                    match: true,
-                },
-                leadingUnderscore: 'allowDouble',
-                trailingUnderscore: 'forbid',
-            },
-        ],
-        'id-blacklist': [2, 'arr', 'cb', 'e', 'el', 'err', 'idx', 'num', 'str', 'tmp', 'val'],
-        'no-return-assign': ['error', 'except-parens'],
-        'no-unused-vars': 'off',
-        '@typescript-eslint/no-unused-vars': [
-            'error',
-            { argsIgnorePattern: '^_', ignoreRestSiblings: true },
-        ],
-
-        'prefer-destructuring': 'warn',
-        'no-nested-ternary': 'warn',
+        'import/no-default-export': 'off',
+        'no-restricted-exports': 'off',
```

Comment:
@kudlajz I'm not yet sure about this one. I suppressed the `no-default-export` rule for now, since enforcing it on our components would require quite a big refactor. With named exports we don't really need the `index.ts` files in components (since we're exporting all of them from `@/components/index` anyway)